### PR TITLE
fix #1331

### LINKED ISSUE #1331: 16.3: Oops! That page doesn’t exist or is private.
image
http://preview.d2l.ai/d2l-en/master/chapter_recommender-systems/mf.html
https://discuss.d2l.ai/t/matrix-factorization/400

### PR DESCRIPTION
**Pull request by StevenJokes**
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-en/PR-1332/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1332/

File: chapter_recommender-systems/mf.md [hunk 1]
```diff
@@ -173,5 +173,5 @@ scores
 
 
 :begin_tab:`mxnet`
-[Discussions](https://discuss.d2l.ai/t/)
+[Discussions](https://discuss.d2l.ai/t/matrix-factorization/400)
```
Comment (astonzhang) [inline chapter_recommender-systems/mf.md:176]: remove "matrix-factorization/"

**Comment by StevenJokes** [comment]
2am now. Just get up from my bed.:cry

**Comment by mli** [comment]
Job d2l-en/PR-1332/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1332/

**Pull request merged**